.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
```

Comment:
The `no-proto` rule is disabled, which allows the use of the `__proto__` property. This can lead to security vulnerabilities and is generally discouraged. Consider enabling this rule by setting it to `"error"` or `"warn"`.